_sections/videocalling-android-studio.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Visual Studio Code
+tab: vscode
+---
+
+## Overview
+
+GitLive's video-calling feature allows you to video call teammates straight from your IDE and work together on files exactly like you would in a Google Doc. This feature will undoubtedly boost your productivity when you want to fix code issues with a teammate.
+
+
+### Instructions
+#### Step 1
+
+ Next to a collaborators name, you will find a phone icon allowing you to place a call with them when clicked, once on the call you can also edit code together.
```

Comment:
Next to a teammate's name, you will find a phone icon allowing you to place a call.